rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -309,17 +313,24 @@ def test_upgrade_v4_v5(globaldb):
     # Check the state before upgrading
     with globaldb.conn.read_ctx() as cursor:
         assert globaldb.get_setting_value('version', None) == 4
+        # check existence of default_rpc_nodes table
         cursor.execute(
             'SELECT COUNT(*) FROM sqlite_master WHERE type="table" and name=?',
             ('default_rpc_nodes',),
         )
         assert cursor.fetchone()[0] == 0
-        last_queried_ts = globaldb_get_general_cache_last_queried_ts_by_key(
+        # check existence of unique_cache table
+        cursor.execute(
+            'SELECT COUNT(*) FROM sqlite_master WHERE type="table" and name=?',
+            ('unique_cache',),
```

Comment:
You have a constant for this, use it
```suggestion
            (UNIQUE_CACHE,),
```